examples/http-circuit-breaker/http_circuit_breaker.bal

```diff
@@ -2,9 +2,9 @@ import ballerina/http;
 import ballerina/log;
 import ballerina/runtime;
 
-// Circuit Breakers are a way of protecting against distributed failure in Ballerina.
-// The circuit breaker is looking for errors across a rolling time window.
-// After breaking the circuit, the circuit breaker will not send any more requests to the backend until the resetTime.
+// Circuit Breakers are used to protect against distributed failure.
```

Comment:
@VijithaEkanayake : Can you take a look at the edits, please. If they are good to go, please approve.